.github/dependabot.yml

```diff
@@ -7,18 +7,9 @@ updates:
     # Check the npm registry for updates every week on monday
     schedule:
       interval: "weekly"
-    reviewers:
-      - "julienw"
-      - "canova"
-      - "gregtatum"
-
-  # Enable version updates for Docker
-  - package-ecosystem: "docker"
```

Comment:
When I configured this, I thought it would be a good idea to update node versions so that we would have newer docker images with the security updates.

But we want to stay on node LTS anyway and upgrade to a new major version only when ready. Updating minor versions can be done by pulling new base images anyway.

So after thinking this more I decided to go with the PR #109 for this. This has the added advantage of pulling security updates for the underlying system even when there's no new node version.

Reply:
Good call. I agree that this dependabot docker config didn't turn out what we want :/ Thanks for removing it.